src/controllers/v1/review-stages.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * name : review-stages.js
+ * author : Priyanka Pradeep
+ * created-date : 29-July-2024
+ * Description : Controller for review stage details.
+ */
+
+const reviewStagesService = require('@services/review-stages')
+const common = require('@constants/common')
+const utils = require('@generics/utils')
+
+module.exports = class reviewStages {
+	/**
+	 * update review stage
+	 * @method
+	 * @name update
+	 * @param {Object} req - request data.
+	 * @returns {JSON} - review stages data
+	 */
+
+	async update(req) {
```

Comment:
@priyanka-TL We don't need create?

Reply:
@aks30  creation is handling in the org config create and update, do we need separate api for create